.github/workflows/java-cd.yml

```diff
@@ -158,3 +143,117 @@ jobs:
                   name: java-${{ matrix.host.TARGET }}
                   path: |
                       java/bundle*.jar
+
+    publish-to-maven-central-deployment:
+        if: ${{ inputs.maven_publish == true || github.event_name == 'push' }}
```

Comment:
Do we need quotes?
```suggestion
        if: ${{ inputs.maven_publish == 'true' || github.event_name == 'push' }}
```

Reply:
No because `maven_publish` is a boolean.